trace_decoder/src/lib.rs

```diff
@@ -104,10 +104,16 @@ use evm_arithmetization::GenerationInputs;
 use keccak_hash::keccak as hash;
 use keccak_hash::H256;
 use mpt_trie::partial_trie::{HashedPartialTrie, OnOrphanedHashNode};
+use plonky2::field::goldilocks_field::GoldilocksField;
+use plonky2::hash::hash_types::NUM_HASH_OUT_ELTS;
 use processed_block_trace::ProcessedTxnInfo;
 use serde::{Deserialize, Serialize};
 use typed_mpt::{StateTrie, StorageTrie, TrieKey};
 
+/// The base field on which statements are being proven.
+// TODO(Robin): https://github.com/0xPolygonZero/zk_evm/issues/531
```

Comment:
Nice :)

nit: I'd go for github username so it's unique, but this works as we're a small team